File: DheerajKumar/problem22.cpp
//22. WAP to input a character from user and check whether it is an alphabets, digits, or special symbols.

#include<iostream>
using namespace std;

void checkCharacter(char userInput){
    int asciiToDecimal = static_cast<int>(userInput);
    if(asciiToDecimal >= 48 && asciiToDecimal <= 57){
        cout<<"Input is a { Number }";
        return;
    }
    if(asciiToDecimal >= 65 && asciiToDecimal <= 90){
        cout<<"Input is a { Capital Alphabet }";
        return;
    }
    if(asciiToDecimal >= 97 && asciiToDecimal <= 122){
        cout<<"Input is a { Small Alphabet }";
        return;
    }
    if(asciiToDecimal >= 127){
        cout<<"Input is a { Special Symbol }";
        return;
    }
    cout<<"Input is a { Symbol }";
    return;
}

int main(){
    char userInput;
    cout<<"Enter the character ->>";
    cin>>userInput;

    checkCharacter(userInput);
}

File: LovelyKumari/BasicProgramming/Problem22.java
import java.util.Scanner;

public class Problem22 {
    //WAP to input a character from user and check whether it is an alphabets, digits, or special symbols
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter a character: ");
        char ch=sc.next().charAt(0);

        // Check if the character is an alphabet, digit, or special symbol

        if(Character.isAlphabetic(ch)){
            System.out.println(ch+" is an Alphabet.");
        } else if(Character.isDigit(ch)){
            System.out.println(ch+" is a Digit.");
        } else {
            System.out.println(ch+" is a Special symbol.");
        }

    }
    
}
